integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -411,6 +406,7 @@ export class CdcJsService implements OnDestroy {
       return this.invokeAPI('accounts.setAccountInfo', {
         ...profileObj,
       }).pipe(
+        take(1),
```

Comment:
Why do we need to put `take(1)` after `invokeAPI`?  AFAIK the `invokeAPI` closes its stream after the first emission OR throws an rxjs error - which closes the stream anyway.

Same question to other places in this PR.

PS. We can move this discussion to a separate PR, as not crucial for 6.0